frontend/src/queries/schema/schema-general.ts

```diff
@@ -3582,12 +3582,24 @@ export interface WebTrendsItem {
     metrics: Partial<Record<WebTrendsMetric, number>>
 }
 
-export interface WebTrendsQueryResponse extends HogQLQueryResponse<WebTrendsItem[]> {
+export interface WebTrendsQueryResponse extends AnalyticsQueryResponseBase {
```

Comment:
Why not keep `extends HogQLQueryResponse` and simply add `results: WebTrendsItem[]` here to override the default?

Also, may I suggest you *remove* the generic from `HogQLQueryResponse` to avoid this from happening again? Ideally we'd add some code to `schema:build` to avoid this from happening tbh

Reply:
Yeah, makes sense to remove the generic. 

The `AnalyticsQueryResponseBase` came before that initial PR was created, and I think it also makes sense to just inherit from it, the reason to add the fields is that it was just the smallest change I could think of to stop this. I have another PR coming that changes the WebTrendsQueryResponse more so it would lower the amount of conflicts as well 😅